src/main/kotlin/blackjack/flow/BlackJackGame.kt

```diff
@@ -0,0 +1,41 @@
+package blackjack.flow
+
+import blackjack.card.Deck
+import blackjack.participant.Dealer
+import blackjack.participant.Participants
+import blackjack.participant.Player
+import blackjack.participant.Players
+import blackjack.view.OutputView
+import blackjack.view.policy.InputMoreCardPolicy
+
+data class BlackJackGame(
+    val participants: Participants,
+    val turnMachine: TurnMachine,
+) {
+    fun play(): Participants {
+        val gamePlayers = participants.gamePlayers
+        val players = gamePlayers.players
+        players.forEach { player -> turnMachine.playerTurn(player) }
+
+        val dealer = participants.dealer
+        turnMachine.dealerTurn(dealer)
+
+        return participants
+    }
```

Comment:
```suggestion
    fun play() {
        val gamePlayers = participants.gamePlayers
        val players = gamePlayers.players
        players.forEach { player -> turnMachine.playerTurn(player) }

        val dealer = participants.dealer
        turnMachine.dealerTurn(dealer)
    }
```

val participants: Participants를 통해서 participants를 가지고 올 수 있기에, 반환은 하지 않아도 될 듯하네요. 이유는 play의 반환 값인 Participants가 불변이 아닌 가변인 이상 BlackJackGame의 프로퍼티로도 충분히 가져올 수 있지 않을까라는 생각을 하였습니다.

Reply:
앗.. 이 부분은 가변이기에 말씀처럼 반환되지 않아도 될 것 같습니다 !

그래도 가장 좋은 방법은 불변으로 관리되는 것이기에 개선 가능하다면 다음 단계 커밋에 이 내용도 포함시켜보겠습니다 😁